docs/index.md

```diff
@@ -4,7 +4,141 @@ hide:
   - toc
 ---
 
-<h1 align="center" style="margin-top: 0px;">Dasharo OSF vPub Summer 2022</h1>
-<h2 align="center" style="margin-top: 0px;">More info coming soon!</h2>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 
-If you are interested in previous events please visit our [archive](archive.md).
+<p align="center" style="margin-bottom: 0px !important;">
+  <img width="800" src="../images/hackathon-2022-poster.png" alt="Hackathon 2022 poster" align="center">
+  </p>
+<br>
+
+---
+
+The main theme of this year's hackathon will be coreboot. coreboot is an
+extended firmware platform that provides instant and secure booting of
+modern computers and embedded systems. As an open source project, it provides
+auditability and maximum control over the technology.
+
+As an active contributor and promoter of open-source firmware, the 3mdeb Team
+wants to familiarize participants with coreboot. You can also participate in
+providing support for the selected platform (more information in the near
+future).
+
+3mdeb will provide the hardware needed to participate in the event. Bring
+your own hardware as well. If we find that it is already supported in the
+coreboot tree, we will be happy to flash coreboot on your device.
+During the event there will be soft drinks and beer, as well as small snacks.
+
+---
+
+<h2 align="left" style="margin-top: 0px;">
+  <b>Calendar Invites:</b>
+  <a href="#">iCal</a> /
+  <a href="#" target="_blank">Google</a>
+</h2>
+<h2 align="left" style="margin-top: 0px;">
+  <b>Schedule:</b>
+  <a href="https://cfp.3mdeb.com/" target="_blank">Hackathon Schedule</a>
+</h2>
+
+---
+
+<h1 align="left" style="margin-top: 0px;">
+Sign up now at:
+<a href="https://cfp.3mdeb.com/hackathon-2022/submit/" target="_blank">https://cfp.3mdeb.com/hackathon-2022/</a>!
+</h1>
+---
+
+<h1 align="left" style="margin-top: 0px;">
+Venue
+</h1>
+
+The event will take place at <a href="https://3mdeb.com/contact/" target="_blank">3mdeb's office</a>, accessible at <a href="https://www.openstreetmap.org/way/103471012#map=19/54.39267/18.58042" target="_blank">Abrahama 1a / 2.05, Gdańsk  80-307</a>.
```

Comment:
Please wrap at 80 characters!

Link to contact page is not needed. Please provide 2 links to location one using openstreet map and other using google maps. Please mention where the links go in parenthesis.

Reply:
Fixed, and added suggestions.